src/main/java/crazypants/enderio/conduit/AbstractItemConduit.java

```diff
@@ -103,6 +104,31 @@ public boolean onItemUse(ItemStack stack, EntityPlayer player, World world, int
             }
           }
           return true;
+        } else {
+          // There's not really any good way to compare conduits...
+          IConduit existingConduit = bundle.getConduit(getBaseConduitType());
+          ItemStack existingConduitAsItemStack = existingConduit.createItem();
+          if (!crazypants.util.ItemUtil.areStacksEqual(existingConduitAsItemStack, stack)) {
+            if (!world.isRemote) {
+              IConduit newConduit = createConduit(stack, player);
+              if (newConduit == null) {
+                System.out.println("AbstractItemConduit.onItemUse: Conduit null.");
```

Comment:
The alternative would be to throw an exception. ;)

The only way that can happen is when there's a bad conduit type. That should be noticed while developing the conduit type, I'd say. This is not a "play time" condition.

And, btw, I just copied the existing code from just above.